flow/Platform.actor.cpp

```diff
@@ -2427,28 +2445,43 @@ std::vector<std::string> findFiles( std::string const& directory, std::string co
 				else
 					continue;
 			}
-			if ((*accept_file)(buf.st_mode, name, extension))
+
+			if ((directoryOnly && acceptDirectory(buf.st_mode, name, extension)) ||
+				(!directoryOnly && acceptFile(buf.st_mode, name, extension))) {
```

Comment:
```suggestion
			    (!directoryOnly && acceptFile(buf.st_mode, name, extension))) {
```

Reply:
For this one, I assume the intent was to align the conditions of the if statement, but if not you can disregard.